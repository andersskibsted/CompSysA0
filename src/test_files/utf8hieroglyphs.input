𓁀	𓁁	𓁂	𓁃	𓁄	𓁅	𓁆	𓁇	𓁈	𓁉	𓁊	𓁋	𓁌	𓁍	𓁎