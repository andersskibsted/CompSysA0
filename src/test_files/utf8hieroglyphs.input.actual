test_files/utf8hieroglyphs.input: data
