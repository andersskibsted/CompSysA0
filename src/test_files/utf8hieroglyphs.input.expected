test_files/utf8hieroglyphs.input: Unicode text, UTF-8 text, with no line terminators
